src/TypeScript.UnitTests/Analyzer/JavaScriptAnalyzerTests.cs

```diff
@@ -367,22 +366,6 @@ public async Task ExecuteAnalysis_ResponseWithIssues_ConsumerCalled()
             consumer.Verify(x => x.Accept("some path", convertedIssues));
         }
 
-        [TestMethod]
-        public async Task ExecuteAnalysis_NonCriticalException_ExceptionCaughtAndLogged()
```

Comment:
This is now handled by the `ExecuteAnalysis_AnalysisFailed_NotifiesThatAnalysisFailed` test.